website/docs.md

```diff
@@ -346,11 +346,13 @@ Listener names must be valid [Go identifiers][identifiers]. For example, the
 names `"foo"`, `"bar42"`, and `"_moo"` are legal, while `""`, `"foo bar"`, and
 `"foo-bar"` are illegal.
 
-Run `go mod tidy` and then `SERVICEWEAVER_CONFIG=weaver.toml go run .`.
+Run `weaver generate`, then `go mod tidy`, and then
+`SERVICEWEAVER_CONFIG=weaver.toml go run .`.
 The program should print out the name of the application and a unique
 deployment id. It should then block serving HTTP requests on `localhost:12345`.
 
 ```console
+$ weaver generate
```

Comment:
I had to run `weaver generate` again before I could run the next commands, so I'm suggesting that this gets called out in the docs.

Reply:
Yup, good spot. I think previously it wasn't needed, but because of some recent changes, it is now needed.